### ISSUE TITLE
WASD keys stop working sometimes

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#365]
Sometimes the WASD keys stop working.

I haven't isolated the exactly when this happens, but have noticed that after jumping, the W key sometimes does not work.

Switching to Arrow keys or jumping again, or releasing and re-grabbing all seem to un-stick the WASD keys.

Marking as low priority for now.

**Comment by jessegreenberg** [comment]
@terracoda please provide platform details if you know them.

**Comment by terracoda** [comment]
@jessegreenberg, I was using VO and Safari on PhET Test version on March 9.

**Comment by jessegreenberg** [comment]
Thanks!

**Comment by terracoda** [comment]
@jessegreenberg, I will add to this issue if I can isolate details as I do other testing. I have noticed this behaviour in previous versions, so it is not new behaviour.

**Comment by jessegreenberg** [comment]
I played with the sim on VoiceOver for 6 minutes, and the WASD and arrow keys worked the entire time. Removing my assignment, if you are able to isolate @terracoda or continue to encounter this please let me know.

**Comment by terracoda** [comment]
Thanks @jessegreenberg, I will try to isolate the issue. By the way did you mix in jumping and try moving up with W?j

E.g. Jump right to Near Wall or to Wall and then press W?

**Comment by jessegreenberg** [comment]
@terracoda Hmm, I can't remember, good question!

**Comment by jessegreenberg** [comment]
I observed this after using the jumping keys. Removing low priority label.

**Comment by jessegreenberg** [comment]
This is platform specific, i cannot reproduce in Chrome or with NVDA with Firefox.

**Comment by terracoda** [comment]
I narrowed down this issue in #386, so deleting this issue.
